tests/fixtures/apidoc/discovery_rule.json

```diff
@@ -0,0 +1 @@
+foreman.json
```

Comment:
you'd need to point this at `luna.json`, otherwise the tests won't know what discovery is.